Lib/platform.py

```diff
@@ -194,7 +194,10 @@ def libc_ver(executable=sys.executable,lib='',version='', chunksize=2048):
         binary = f.read(chunksize)
         pos = 0
         while pos < len(binary):
-            m = _libc_search.search(binary,pos)
+            if 'libc' in binary or 'GLIBC' in binary:
```

Comment:
Alternate suggestion:
```suggestion
            if pos or 'libc' in binary or 'GLIBC' in binary:
```

Reply:
I don't see the point of avoiding the two "in" if pos==0? Does it provide any speedup?

This code comes from the master branch. I have have a clever optimization, maybe write it in the master branch first, no?

This change already makes the function 16x faster, it should be enough no?

Reply:
It avoids two "in" if `pos != 0`.

If `pos == 0`, we test just read block. In common case it doesn't contain 'libc', so this optimization makes sense. If `pos != 0`, then 'libc' was already found in this block, so `'libc' in binary` will be always true, and performing this test just wastes a time.